### ISSUE TITLE
tx shows duplicate after a few second in the wallet tab

### ISSUE THREAD

**Issue by michieldespiegelaere** [LiskHQ/lisk-hub#1191]
### Expected behaviour
Tx only showing once in the wallet while it's sending.

### Actual behaviour
While the tx is sending it duplicates after a few seconds. (you can see it with the address 4560518704624832050L in the picture).

### Steps to reproduce
Make a tx in lisk hub 1.1.0-rc.1.

![image](https://user-images.githubusercontent.com/17978514/44288756-a64f0200-a271-11e8-8386-f5b7cfe4b6ea.png)

**Comment by faival** [comment]
Hi @michieldespiegelaere , I just reproduced steps in hub 1.1.0-rc.1 but I was unable to see the duplicate transaction effect. 

Can you please perform a `Ctrl + cmd + i`, while on hub electron app,  and provide version of hub which is displayed in the head of the document?

This way we can narrow down if there was a problem with the install of the new version. 

```
<meta name="application-name" content="lisk-hub-1.1.0-rc.1" data-lisk-hub-version="1.1.0-rc.1">

```

**Comment by michieldespiegelaere** [comment]
Hi @faival, I've just tried to reproduce it showed up less than  friday night. I tried it to multiple addresses, with different amounts of lisk and with or without a reference.

this is what I find in the head of the document.
`<meta name="application-name" content="lisk-hub-1.1.0-rc.1" data-lisk-hub-version="1.1.0-rc.1">`

![image](https://user-images.githubusercontent.com/17978514/44339620-5b580900-a482-11e8-8ef4-675d031959c8.png)

the tx with 12 lisk to 6121750279903659323L showed duplicate just before it was send.

**Comment by faival** [comment]
Hi @michieldespiegelaere , it's a bit strange, the screenshot you are showing is loading a bundle with different hash than for `1.1.0-rc.1`,

Hub on `1.1.0-rc.1` it's loading the following bundle hashes:

- build/bundle.app.9caf9a3b5bd1139337e4.js
- build/bundle.vendor.9caf9a3b5bd1139337e4.js

**Comment by michieldespiegelaere** [comment]
@faival I see, could this be the problem?
my bundle hashes are 
<ul>
<li>build/bundle.app.c202b256f52d65551ce6.js</li>
<li>build/bundle.vendor.c202b256f52d65551ce6.js</li>
</ul>

Should I retry downloading `1.1.0-rc.1` and look if my bundle hashes are different?

**Comment by faival** [comment]
@michieldespiegelaere yes please, could you try installing rc.1 from scratch, and may be renaming any previous version installed?

This way we can isolate the issue to a wrong bundle hash  (maybe due to the electron update process), and therefore not loading the rc.1 source code base.

**Comment by michieldespiegelaere** [comment]
@faival @slaweet I think that's correct about the bundle hash.

I've installed `1.1.0-rc.1` on 3 devices and they all give the same bundle hash that I posted here.

**Comment by faival** [comment]
Hi, @michieldespiegelaere , I booted hub 1.1.0-rc.1 in a windows 10 with VirtualBox. 

I was not able to reproduce the effect of duplicate transaction. 

transaction is added as pending (with three dots)
after block updated and validated, transaction shows in the list as confirmed (not three dots and date available. )
there are no duplicated transactions. 

see screenshot for version and bundle hash reference. 

![image](https://user-images.githubusercontent.com/668295/44906864-6b8ba600-ad16-11e8-94c5-4d1f052b94c5.png)

**Comment by michieldespiegelaere** [comment]
Hi @faival I've just send a tx with hub 1.1.0 and I didn't see the duplication. So we can say that this is resolved?
